### ISSUE TITLE
add support for exporting & importing the app settings

### ISSUE THREAD

**Issue by 07416** [markusfisch/BinaryEye#330]
There's no such feature, as far as I can see.

**Comment by sorryusernameisalreadytaken** [comment]
I would love to see that feature too. There is just a chance to export the history of all qr-scans. But not importing and no export/import of settings too.

**Comment by Mobasheera** [comment]
I would love to see this feature too!! I have recently performed a factory reset on my device and my history is lost! I have the exported files but there's no import option in the app...

**Comment by FoxP** [comment]
+1, learnt it the hard way (my phone randomly died some days ago). Lost everything.

**Comment by nlsdg** [comment]
You can import/export settings via the profiles. But that doesn't export the scan history.

You can share the history via the share button or the Export button in multiple format.
However, indeed still no function to import history (on version 1.75)

**Comment by Mobasheera** [comment]
Yeah correct, still no Import/Export the entire history feature in the app
